app/models/product.rb

```diff
@@ -209,8 +209,8 @@ def create_default_price_group_products
     end
   end
 
-  def create_nonbillable_price_policy
-    PricePolicyBuilder.create_nonbillable_for(self)
+  def create_skip_review_price_policies
```

Comment:
I'm not sure about this - maybe it makes sense to have different logic for Skip Review vs Nonbillable.